<a href="https://colab.research.google.com/github/LuisaMariaMontoyaConde1994/TrabajoClaseestadistica/blob/main/Actividad_01.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

***Luisa Fernanda Sánchez Herrera***

En una elección, un candidato tiene una probabilidad del $60\%$ de recibir un voto en cada votación individual.

Si se emiten $20$ votos.

1.   ¿Cuál es la probabilidad de que al menos obtenga $12$ votos?
2.   ¿Cuál es la probabilidad de que a lo más obtenga $5$ votos?
3.  ¿Cuál es la probabilidad de que no obtenga votos?
4.  ¿Cuál es la probabilidad de que obtenga entre $8$ y $10$ votos?

¿Cuál es la probabilidad de que al menos obtenga $12$ votos?

$$P(X≥12)=1-P(X<12)$$

In [ ]:
import numpy as np
import matplotlib.pyplot as plt
from scipy.stats import binom

In [ ]:
n=20
p=0.6
x=12
prob=1-binom.cdf(x-1,n,p)
print(prob)
m=prob*100
print('La probabilidad es',m,'%')

0.5955987253122248
La probabilidad es 59.55987253122248 %


***Rta:*** La probabilidad de que al menos obtenga $12$ votos es del  $59.55\%$  aproximadamente.

¿Cuál es la probabilidad de que a lo más obtenga  $5$  votos?

$$P(X\leq 5)$$

In [ ]:
n=20
p=0.6
x=5
prob=binom.cdf(x,n,p)
print(prob)
m=prob*100
print('La probabilidad es',m,'%')

0.001611524643456288
La probabilidad es 0.1611524643456288 %


***Rta:*** La probabilidad de que a lo más obtenga $5$ votos es del  $0.16\%$  aproximadamente.

¿Cuál es la probabilidad de que no obtenga votos?

$$P(X=0)=\binom{20}{0}(0.6)^{0}(0.4)^{20-0}$$
$$≈1.0995116277760013e-08$$
$$≈1.0995116277760013e-08(100\%)$$
$$=1.09e\%$$

In [ ]:
n=20
p=0.6
x=0
prob=binom.pmf(x,n,p)
print(prob)
m=prob*100
print('La probabilidad es',m,'%')

1.0995116277760013e-08
La probabilidad es 1.0995116277760013e-06 %


***Rta:*** La probabilidad de que no obtenga votos es del  $1.09e\%$  aproximadamente

¿Cuál es la probabilidad de que obtenga entre  $8$  y  $10$  votos?

$$P(8\leq P(x) \leq 10)$$

In [ ]:
n=20
p=0.6
x=10
prob1=binom.cdf(x,n,p)
print(prob1)
m=prob1*100
print(m,'%')

0.24466279668360685
24.466279668360684 %


In [ ]:
n=20
p=0.6
x=8
prob2=binom.cdf(x,n,p)
print(prob2)
t=prob2*100
print(t,'%')

0.056526367034253025
5.652636703425302 %


In [ ]:
r=m-t
print('La probabilidad es',r,'%')

La probabilidad es 18.813642964935383 %


***Rta:*** La probabilidad de que obtenga entre $8$ y $10$ votos es del  $18.81\%$  aproximadamente